examples/custom/src/registerServiceWorker.js

```diff
@@ -0,0 +1,108 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
you can :zap: this file and all other service worker files from our examples (and of course the import of these files). They're only useful in production environments.

Reply:
Agreed.